common/messages/gameStart.ts

```diff
@@ -0,0 +1,19 @@
+export interface DownEvents {
+  // updateScoreboard: (data: Array<{ color: string, hex: number, points: number }>) => void;
+  // endSequence: (data: Array<{ color: string, hex: number, points: number }>) => void;
+  // startSequence: () => void;
+  // showVotingSequence: (votingSequence: string) => void;
+  // hideVotingSequence: () => void;
+  // sendVotingCountdown: (secondsLeft: number) => void;
```

Comment:
Are we able to delete the commented-out lines from this file? I noticed that you included them in the other new fiels.